x11-misc/xbattbar/xbattbar-1.4.9.ebuild

```diff
@@ -2,6 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
+
 inherit flag-o-matic multilib toolchain-funcs
```

Comment:
Drop multilib? It’s not needed for much in newer EAPIs, common mistake is to use it for libdir in newer ones

Reply:
Yeah, good catch, dropped